packages/app-builder-backend/package.json

```diff
@@ -10,29 +10,34 @@
   "author": "Reapit Ltd",
   "main": "./src/index.ts",
   "scripts": {
-    "start": "ts-node .",
+    "start": "DYNAMODB_ENDPOINT=http://localhost:8000 ts-node .",
```

Comment:
config env?

Reply:
i would but because the lambda's using environment variables when deployed this felt cleaner